docs/learning/transformer/branching-imodels.md

```diff
@@ -0,0 +1,220 @@
+# iModel Transformation and iModel Branching
+
+## What is branching
+
+It can be useful in some applications to take iModels and their linear concept of history, and introduce the concept of
+*branching*, where an iModel diverges into two separate iModels after a changeset, allowing each to have new distinct changes and
+a distinct history. Typically, one of them is referred to as the *master iModel*, and the other the *branch iModel*,
+with the master iModel being the one that existed before the branch was created from it.
+Importantly, these branches can often be used to updates to one branch iModel while the master iModel
```

Comment:
I think you're missing a "make" before "updates" and the next line it should be "receive updates" instead of "received".

Reply:
please resolve if adequate

Reply:
@calebmshafer could you take another look at this?

Reply:
I replied to this comment when my page was out of date, sorry :)